lib/broccoli/broccoli-config-replace.js

```diff
@@ -4,68 +4,68 @@ var fs = require('fs-extra');
 var path = require('path');
```

Comment:
its still there. Maybe a PR didnt get merged in? https://github.com/angular/angular-cli/blob/master/lib/broccoli/broccoli-config-replace.js